README.md

```diff
@@ -300,6 +320,43 @@ A:  If you know of a better way to do something I am attempting in this plugin,
     me or make the changes yourself and open a pull request. If I am doing something that is bad
     or can be improved, I am more than willing to hear about it!
 
+## Deprecated commands
```

Comment:
Probably useful to also include a section for these in the docs just to note they're deprecated and point to the options that should be used instead.